sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -919,6 +919,8 @@ object FunctionRegistry {
   registerInternalExpression[EWM]("ewm")
   registerInternalExpression[NullIndex]("null_index")
   registerInternalExpression[CastTimestampNTZToLong]("timestamp_ntz_to_long")
+  registerInternalExpression[FromProtobuf]("from_protobuf")
+  registerInternalExpression[ToProtobuf]("to_protobuf")
```

Comment:
may I ask why not using the public `from_protobuf` and `to_protobuf`?

https://github.com/apache/spark/blob/ab3cd910e0c9db1ea82da65ef6ba7338a2f128bb/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala#L876-L878

Reply:
Yeah, I was wondering the same.

Reply:
I added this since the `ProtobufFunctionsSuite` would fail as below without registering internal expression here:

```
org.apache.spark.sql.AnalysisException: [UNRESOLVED_ROUTINE] Cannot resolve routine `from_protobuf` on search path [`system`.`builtin`, `system`.`session`, `spark_catalog`.`default`]. SQLSTATE: 42883
```

Is there happen to any way to handle this without registering internal expression?

Reply:
are some constructors missing?

Reply:
Ah, I misunderstood that we need both public and internal protobuf functions for different purpose but just realized that we can just reuse the existing one. Updated to use the existing public ones